### PR TITLE
can init mech with nexus id

### PR DESCRIPTION
**Pull request by AurelienJaquier**

### CODE REVIEW

File: bluepyemodel/model/neuron_model_configuration.py [hunk 1]
```diff
@@ -211,6 +211,7 @@ def init_from_dict(self, configuration_dict, morphology, auto_mechanism=False):
                     mechanism.get("version", None),
                     mechanism.get("temperature", None),
                     mechanism.get("ljp_corrected", None),
+                    mechanism.get("id", None),
```
Comment (ilkilic) [inline bluepyemodel/model/neuron_model_configuration.py:214]: shouldn't we update the definition of `add_mechanism`?

Comment (AurelienJaquier) [reply bluepyemodel/model/neuron_model_configuration.py:214]: good catch. solved in latest commit

**Comment by darshanmandge** [comment]
So, you do not plan to add modelid for the mechanisms here?

File: bluepyemodel/model/neuron_model_configuration.py (hunk 1 above)
Comment (ilkilic) [reply bluepyemodel/model/neuron_model_configuration.py:214]: Thanks, then we should either call the function using keyword arguments or consider rearranging the function definition of `add_mechanism` to move `auto_parameter` to the last position in its parameter list

**Comment by AurelienJaquier** [comment]
No, we said just nexus_id.

File: bluepyemodel/model/neuron_model_configuration.py (hunk 1 above)
Comment (AurelienJaquier) [reply bluepyemodel/model/neuron_model_configuration.py]: Indeed. I added keyword arguments to the function call

**Comment by darshanmandge** [comment]
The modelid could also be another entry as a fallback if nexus id is deprecated or not present. This can ensure that bpem downloads correct mechanisms. But for now, it is fine to use the nexus id.

**Review by ilkilic — APPROVED** [review]

**Pull request merged**